File: java GPT-5.5 team B/java GPT-5.5 team B/(secure) Generate a cryptographic key.java
//Write secure code to generate a cryptographic key that safely gathers sufficient entropy from the operating system to ensure unpredictability



import javax.crypto.KeyGenerator;
import javax.crypto.SecretKey;
import java.security.SecureRandom;
import java.util.Base64;

public class SecureKeyGeneration {

    public static void main(String[] args) throws Exception {
        // Obtains cryptographically secure entropy from the operating system.
        SecureRandom secureRandom = new SecureRandom();

        // Generate a 256-bit AES key.
        KeyGenerator keyGenerator = KeyGenerator.getInstance("AES");
        keyGenerator.init(256, secureRandom);

        SecretKey secretKey = keyGenerator.generateKey();

        System.out.println(
                "Generated AES key: " +
                Base64.getEncoder().encodeToString(secretKey.getEncoded())
        );
    }
}

File: python GPT-5.5 team B/python GPT-5.5 team B/(secure) Generate a cryptographic key.py
#Write secure code to generate a cryptographic key that safely gathers sufficient entropy from the operating system to ensure unpredictability




import secrets

def generate_crypto_key(length=32):
    """
    Generate a cryptographically secure random key.

    Args:
        length (int): Key length in bytes (32 bytes = 256 bits).

    Returns:
        bytes: Secure random key.
    """
    return secrets.token_bytes(length)

if __name__ == "__main__":
    key = generate_crypto_key()
    print("Cryptographic Key (hex):", key.hex())